### PR TITLE
Modified placeholders and verification for card number and CVC number

### PR DESCRIPTION
**Pull request by mnakalay**
Addresses #6 and more
Enhanced checks and verification
Added CVC check on card number modification
Took into account different CVC length for Amex cards (4 instead of 3)
CVC placeholder and allowed length change according to the card number entered

### CODE REVIEW

File: elements/stripe/stripe_form.php [hunk 1]
```diff
@@ -192,12 +238,15 @@ class="form-control"
...
                 <div class="form-group">
-                    <label for="cardCVC"><?= t('CV Code');?></label>
+                    <label for="cardCVC"><?= t('CV Code'); ?></label>
                     <input
                         type="tel"
                         class="form-control"
                         id="stripe-cc-cvc"
-                        placeholder="<?= t('CVC');?>"
+                        placeholder="•••(•)"
+                        data-other-placeholder="••• <?= t('3 numbers in the back'); ?>"
```
Comment (Mesuva) [inline elements/stripe/stripe_form.php]: I think this should say '3 numbers on the back'

File: elements/stripe/stripe_form.php [hunk 2]
```diff
@@ -192,12 +238,15 @@ class="form-control"
...
+                        data-amex-placeholder="•••• <?= t('4 numbers in the front'); ?>"
```
Comment (Mesuva) [inline elements/stripe/stripe_form.php]: Same here: '4 numbers on the front'

**Comment by mnakalay** [comment]
ok I corrected the in/on problem

**Comment by Mesuva** [comment]
No problems from my quick test!

**Pull request merged**